src/c20_server/job_queue.py

```diff
@@ -0,0 +1,30 @@
+
+"""
+implementation of Job_Queue class
+
+"""
+
+
+class JobQueue:
+    """
+    Job Queue class.
+    """
+    unsigned_jobs_list = []
+
+    def add_job(self, job_id):
```

Comment:
We need to store objects, not ints.

Make `Job` class that encapsulates this int value.